src/debug.rs

```diff
@@ -51,6 +60,19 @@ pub enum ValidationError {
     RetentionBelowMinimum,
 }
 
+/// Export mode for the debug bundle.
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum BundleExport {
+    /// Bundle export is disabled entirely.
+    Disable,
```

Comment:
It looks like `BundleExport::Disable` is only used in tests right now. Is this something we want to allow users to turn on via config? I can't really think of a reason for this, but wanted to double check

Reply:
I don't think there's any reason to allow users to set this.